src/main/java/net/ossindex/gradle/OssIndexPlugin.java

```diff
@@ -22,20 +23,41 @@
 
 public class OssIndexPlugin implements Plugin<Project> {
 
+    public static JunitXmlReportWriter junitXmlReportWriter = null;
+
+    public static String junitReport = null;
+    public static Integer instanceId;
+
+    private static AuditExtensions settings = null;
+
     private static final Logger logger = LoggerFactory.getLogger(OssIndexPlugin.class);
     private List<Proxy> proxies = new LinkedList<>();
 
     private AuditorFactory factory = new AuditorFactory();
+    private AuditResultReporter reporter = null;
 
     public void setAuditorFactory(AuditorFactory factory) {
         this.factory = factory;
     }
+    private Project project = null;
+
+    public OssIndexPlugin() {
+        if (this.junitXmlReportWriter == null) {

```

Comment:
I made a comment here earlier about the check for null not being required. I didn't note that it was a static variable. It *is* required.